CUU/Classes/PersonaKit/PKSession.swift

```diff
@@ -177,7 +177,3 @@ extension PKSession {
         return numberOfShortVisits > 0
     }
 }
-
-extension PKSession: Identifiable {
```

Comment:
@ffittschen Can you confirm that this can be deleted safely?

Reply:
@jpbernius Yes, seems like it was unused

Reply:
I actually can't remember why the `/Storage` was added in the first place

Reply:
Thanks for your Feedback @ffittschen! I just removed `PersonaKit/Storage/*` in e91e057 and it still builds, so I guess this is fine.